tests/test_field.py

```diff
@@ -48,6 +48,24 @@ def test_boolean_deserialization() -> None:
     assert bf.deserialize(1)
 
 
+def test_datetime_deserialization() -> None:
+    f = field.Date()
+    dt = datetime.now()
+    assert dt == f._deserialize(dt.isoformat())
+
+    d = date.today()
+    assert datetime.combine(d, time()) == f._deserialize(d.isoformat())
```

Comment:
It took me a while to understand that this wasn't time.time() but converting to a datetime at midnight.

Reply:
I had a passing thought that `datetime.time()` might be confused with `time.time()` as well. Maybe I should namespace it here.